README.md

```diff
@@ -23,7 +23,7 @@ var options = {
   cssDir: "assets/css",
   fullException: false
 }
-var outputTree = compileSass(inputTrees, options);
+var outputTree = new compileSass(inputTrees, options);
```

Comment:
@eoneill This looks funny (class names should be capitalized.) I’d like to see the `new` keyword be optional, as discussed in this PR’s comments